### PR TITLE
Package 1 1 1

### PR DESCRIPTION
**Pull request by mkoistinen**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -22,11 +22,14 @@
         'Topic :: Internet :: WWW/HTTP',
     ],
     install_requires=[
+        # IMPORTANT: If the minimum version of Django or django Filer changes,
+        # please also update cmsplugin_filer_utils.requirements_check()
```
Comment (czpython) [inline setup.py]: not applicable atm

**Comment by czpython** [comment]
👍

**Comment by stefanfoulis** [comment]
👍

File: HISTORY [hunk 2]
```diff
@@ -1,14 +1,23 @@
-Changelog for django-filer-cmsplugins
-=====================================
+CHANGLOG
```
Comment (yakky) [inline HISTORY]: Changelog

**Pull request merged**